ui/pages/confirmations/hooks/alerts/transactions/useNonContractAddressAlerts.ts

```diff
@@ -43,7 +45,10 @@ export function useNonContractAddressAlerts(): Alert[] {
     !isContractDeploymentTx;
 
   return useMemo(() => {
-    if (!isSendingHexDataWhileInteractingWithNonContractAddress) {
+    if (
+      !isSendingHexDataWhileInteractingWithNonContractAddress ||
+      isUpgradeTransaction
```

Comment:
Should we add this to the dependencies? Or combine them into a `showAlert` boolean?